### PR TITLE
Feature/add motorsport magazin

### PR DESCRIPTION
**Pull request by lsch0lz**
Hi!
I added the MotorSport Magazin [LINK](https://www.motorsport-magazin.com/) to fundus 🚀

Note: Black reformatted several files, that I didn't intent to touch. Let me know, if thats OK for you

### CODE REVIEW

File: src/fundus/publishers/de/motorsport_magazin.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class MotorSportMagazinParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector: CSSSelector = CSSSelector("section.article-body")
```
Comment (addie9800) [inline src/fundus/publishers/de/motorsport_magazin.py:16]: This selector is a bit too coarse, the whole text is extracted as a whole blob. 
```suggestion
        _paragraph_selector: CSSSelector = CSSSelector("section.article-body > p")
```

File: src/fundus/publishers/de/motorsport_magazin.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        _subheadline_selector: CSSSelector = CSSSelector("p.teaser")
```
Comment (addie9800) [inline src/fundus/publishers/de/motorsport_magazin.py:17]: I see the potential for the misunderstanding, but in Fundus this part of the article is the summary, so it should be the _summary_selector (and also updated within the body() function)
But then this leaves a space for the actual subheadline selector. This article is a good example for that: https://www.motorsport-magazin.com/formel1/news-288319-nico-huelkenberg-wechselt-zu-sauber-audi-sichert-sich-deutsche-formel-1-speerspitze/ . I would suggest something along the lines of `section.article-body > h2`

File: src/fundus/publishers/de/motorsport_magazin.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+        @attribute
```
Comment (addie9800) [inline src/fundus/publishers/de/motorsport_magazin.py:19]: Could you also add a function for the topics? Motorsport Magazin supports the meta `news_keywords` tag which could be used for that purpose.

**Review by addie9800 — COMMENTED** [review]
Thanks a lot for adding the motorsport magazin. Be sure to run `black .` before your final commit, otherwise the tests won't go through :)

File: src/fundus/publishers/de/motorsport_magazin.py (hunk 2 above)
Comment (lsch0lz) [reply src/fundus/publishers/de/motorsport_magazin.py]: added!

File: src/fundus/publishers/de/motorsport_magazin.py (hunk 3 above)
Comment (lsch0lz) [reply src/fundus/publishers/de/motorsport_magazin.py:21]: also added! :)

**Comment by lsch0lz** [comment]
Hi! @addie9800 
I added the correct summary and subheadline selectors. Also the class now has a ```topics()``` method.

**Comment by addie9800** [comment]
> Hi! @addie9800 
> I added the correct summary and subheadline selectors. Also the class now has a ```topics()``` method.

Thanks a lot ;) this looks really good. Black seems really weird, when I run black locally it reformats the file back. But I'll just approve the workflow and we'll see if the test go through.

**Comment by lsch0lz** [comment]
Sounds good!

**Comment by MaxDall** [comment]
@lsch0lz Could you tell me what version of black is installed in your environment?

@addie9800 The tests won't trigger. The last commit on this branch 29c4a5c3f1e1bfc67704bb93ff41c8f6044f38d5 was auto-committed and due to GitHubs limitations, those won't trigger workflow runs.

**Comment by lsch0lz** [comment]
@MaxDall I have the newest black version (24.4.2) installed. Just checked and ```black .``` doesn't reformat any of the code locally

**Comment by MaxDall** [comment]
@lsch0lz Thanks :) We pinned the `Black` version for the repository and are currently at `23.1.0`. I'm gonna reformat with `23.1.0` and push on your branch. Then this should be good to go 👍

**Review by MaxDall — DISMISSED** [review]

**Pull request merged**